data-prepper-plugins/in-memory-source-coordination-store/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id 'java'
+}
+
+dependencies {
+    implementation project(':data-prepper-api')
+    testImplementation platform('org.junit:junit-bom:5.9.1')
+    testImplementation 'org.junit.jupiter:junit-jupiter'
+}
+
```

Comment:
Can you add a code coverage level requirement to this plugin?

Reply:
Will do